activities/community-calls/index.md

```diff
@@ -1,5 +1,7 @@
 ---
 type: Index
+redirect_from:
+    - activities/codebase-stewardship/community-calls
 ---
 
 # Community calls
```

Comment:
This paragraph is incomplete/confusing now, since they are not split up into two different set of instructions any more..